jujugui/static/gui/src/app/app.js

```diff
@@ -823,19 +804,17 @@ YUI.add('juju-gui', function(Y) {
         console.log('controller connected');
         const creds = this.controllerAPI.getCredentials();
         const gisf = this.get('gisf');
-        if (!creds.areAvailable && !gisf) {
```

Comment:
And moved it back down after I remembered why I had moved it in the first place :) 

We need a catch-all if no credentials are provided to show the login so that it works across all of the various systems.